### ISSUE TITLE
docs: clarify form control usage in form-field w/ chip-list and chip-input

### ISSUE THREAD

**Issue by btxtiger** [angular/components#19118]
#### Reproduction

Use StackBlitz to reproduce your issue:
https://stackblitz.com/edit/angular-b8qqaf

Steps to reproduce:
1. Create chip list input 
https://material.angular.io/components/chips/overview
2. Disable form field
 
 
#### Expected Behavior

What behavior were you expecting to see?
Should be disabled, light gray color

#### Actual Behavior

What behavior did you actually see?
Is functionally disabled, but not visibly

#### Environment

- Angular: 9
- CDK/Material: 9
- Browser(s): Edge chromium
- Operating System (e.g. Windows, macOS, Ubuntu):  macOS

**Comment by btxtiger** [comment]
It seems `mat-form-field-disabled` ist not added for chip-list form-fields, and form-field ist still focusable

**Comment by devversion** [comment]
The problem here is that the `MatChipList` is the actual form-field control. The form control state of the input is not propagated to the chip list, nor to the parent form-field.

A fix would be to capture the disabled state of the input in the chip list, but conceptually it should be the other way around. i.e. the chip-list is the primary source of data, and notifies the chip input if it should be disabled. The input could be disabled, while the chip list is not.

We should probably clarify this in the documentation. Happy to chat more about it though.

**Comment by btxtiger** [comment]
@devversion 
Indeed, i just moved the `[formControl]="fruitControl"` from `input` to `mat-chip-list` and disabling works properly. On the other hand adding `[disabled]="fruitCtrl.disabled"` to `mat-chip-list` also works. So this is not a bug. 

Maybe the example in the docs could have a correctly placed `formControl` directive or `[(ngModel)]`, to make it obvious. I did not realize that `mat-chip-list` could be a form field, so I implemented a class member array to manage the chip-list data.
